styles/styles.css

```diff
@@ -25,6 +25,7 @@
   --primary-blue-100: #012740;
   --primary-blue-101: #002035;
   --primary-blue-102: #001C2D;
+  --primary-blue-103: #00538A;
```

Comment:
@thomsebastin  are we ok with this variable ?

Reply:
I wouldn't categorize it with this group but name it separately. Anyway, the client has requested to completely use their variables from design system going forward. We will be replace all of these with that. Let's merge this for now and make changes in that PR. @mkbansal1